.github/workflows/ci-cd.yml

```
@@ -108,28 +108,36 @@ jobs:
+      with:
+        repository: UruruLab/Ururu-AI-Config
+        path: config
+        token: ${{ secrets.PRIVATE_REPO_TOKEN }}
+        
+    - name: Copy Development Environment Config Files
       run: |
-        printf 'SPRING_BOOT_BASE_URL=%s\n' "${{ secrets.DEV_SPRING_BACKEND_URL }}" >> .env.development
-        printf 'REDIS_URL=%s\n' "${{ secrets.DEV_REDIS_URL }}" >> .env.development
+        cp config/.env.development .env.development
+        echo "✅ Development environment config files copied successfully"
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)